pkgs/by-name/cr/crates-lsp/package.nix

```
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "crates-lsp";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "MathiasPius";
+    repo = "crates-lsp";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9+0qgdUn5l9oQQavbnR6rHe5zp5WHhGuRyOXt2Dv8Tw=";
+  };
+
+  cargoHash = "sha256-oS6xi8BH5vCVOimYWsDoW0Na7eUXzeHKKSOwpK9wbu8=";
+
+  checkFlags = [
```

```suggestion
  checkFlags = lib.optionals (with stdenv.buildPlatform; isx86_64 && isDarwin) [
```

I was wondering which of `buildPlatform` or `hostPlatform` is more appropriate. This would obviously only make a difference for cross builds, but I’d expect the issue to originate from the host running the tests, making `buildPlatform` the right choice.